launch_crab_multithreading.py

```diff
@@ -1240,10 +1250,11 @@ def submit(config, dryrun = False):
             #FIXME JERstring should not be needed anymore. Test miniAOD ntuplizer without this parameter!
             #FIXME isCentralProd is not yet implemented in AOD ntuplizer. Add!
             #FIXME Once those two parameters work similarly for AOD and miniAOD, remove the if...else below
```

Comment:
Nice! One thing less to fix :)

But this reminds me we have some cleaning up to do: We had discussed the idea of keeping only task-dependent parameters here and moving all other strings to the respective ntuplizers. 

The main reason this wasn't done yet was the run-dependent SF for single-muon triggers in 2018A, which haven't been implemented yet. So far, we just print a warning for every task using 2018 samples. I think one potential solution could be having two additional json files (with runs before & after the muon HLT upgrade) so we can submit separate tasks with different SF filenames if/when needed. This is still not urgent to fix, but should be simple enough to implement in one of the upcoming PR's. Opinions?